pages/resetpassword.tsx

```diff
@@ -60,30 +57,27 @@ export default function Login() {
             }
         })
             .then((res) => {
-                event?.target.reset();
+                reset();
 
                 if (res.status === 200) {
                     return setAxiosSuccessMessage(
                         'You will receive a SMS from MySejahtera to reset your password shortly'
                     );
+                } else {
+                    // TODO: add sentry logging
                 }
 
                 return setAxiosSuccessMessage(res.data);
             })
             .catch((err) => {
-                if (err.response.status === 418) {
-                    return setAxiosErrorMessage('Hmm Server hang... Please Try again ');
-                }
-                event?.target.reset();
```

Comment:
Should we reset the form when there is an error ? 
my purpose of this line of code is to catch axios error from the front end.